package.json

```diff
@@ -47,5 +48,6 @@
   "peerDependencies": {
     "react": ">=16.0.0",
     "react-dom": ">=16.0.0"
-  }
+  },
+  "typings": "index"
```

Comment:
```suggestion
  "types": "index.d.ts"
```

Reply:
I think typings and types are synonymous(?), but I'm more than happy to oblige :)

Reply:
Doesn't really matter, just the extension might be important

Reply:
Ahh, I see, I overlooked the extension bit. Pushed an update.